app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java

```diff
@@ -678,7 +678,25 @@ public void setTitle(String title) {
 
     private String getRunTime() {
         Long runtime = Utils.getSafeValue(mBaseItem.getRunTimeTicks(), mBaseItem.getRunTimeTicks());
-        return runtime != null && runtime > 0 ? String.format("%d%s", (int) Math.ceil((double) runtime / 600000000), getString(R.string.lbl_min)) : "";
+
+        if (runtime == null || runtime <= 0) {
+            return "";
+        }
+
+        int totalMinutes = (int) Math.ceil((double) runtime / 600000000);
+
+        int hours = totalMinutes / 60;
+        int minutes = totalMinutes % 60;
+
+        String runtimeString = "";
+
+        if (hours > 0) {
+            runtimeString += hours + getString(R.string.hours_shorthand) + " ";
```

Comment:
Use string formatting instead so translators can place the digit behind the shorthand if their language needs that. Something like `%1$dh %1$dm` for the hour+minute string should work.

Reply:
Ah, I see. Thanks for the feedback. Changes completed as requested.